packages/core/components/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../tsconfig.json",
```

Comment:
Each package's tsconfig.json extends from the tsconfig.json in the root of the project. Still, there are properties that we have to include here which cannot be included once at the root.